Form/Handler/AuthorizeFormHandler.php

```diff
@@ -41,13 +47,15 @@ public function isRejected()
 
     public function process()
     {
+        $this->request = $this->requestStack->getCurrentRequest();
+
         $this->form->setData(new Authorize(
             $this->request->request->has('accepted'),
             $this->request->query->all()
         ));
 
         if ('POST' === $this->request->getMethod()) {
-            $this->form->bind($this->request);
+            $this->form->submit($this->request);
```

Comment:
Passing Request to submit is now deprecated, you need to swap this to handleRequest. 

In this case it doesn't matter, but if elsewhere you had to change submit for handleRequest and the method of Request could be PUT, you need to ensure that the form object was created with $options --> ['method', 'PUT'].